### PR TITLE
#21711 Show a special note pointing that items are filtered

### LINKED ISSUE #21711: full table list
### Description

Only few tables are displayed on treeview. Refresh not do not change anything (on Tables or on Db).  
The shema have 198 tables, only 5 was listed.

image

### DBeaver Version

Community Edition 22.3.5.202302180832

### Operating System

max Os Ventura 13.4

### Database and driver

Database Postgres 13.11

### Steps to reproduce

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ShadelessFox**
|Default look|When hovered|
|---|---|
|image|image|

### CODE REVIEW

**Review by LonwoLonwo — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/database/DatabaseNavigatorLabelProvider.java [hunk 1]
```diff
@@ -258,6 +262,8 @@ public String getToolTipText(Object element) {
                 return description;
             }
             return ((DBNNode) element).getNodeName();
+        } else if (element instanceof IToolTipProvider) {
+            return ((IToolTipProvider) element).getToolTipText(element);
```
Comment (flambchap) [inline plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/database/DatabaseNavigatorLabelProvider.java]:
```suggestion
        } else if (element instanceof IToolTipProvider toolTipProvider) {
            return toolTipProvider.getToolTipText(element);
```

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/database/DatabaseNavigatorLabelProvider.java]: Uh, not sure how I missed that

**Comment by MashaKorax** [comment]
redundant copy option in context menu
dbeaver_pH97ezQ9Iv

**Comment by MashaKorax** [comment]
verified

**Pull request merged**